pkg/agent/controller/traceflow/traceflow_controller.go

```diff
@@ -318,6 +318,14 @@ func (c *Controller) injectPacket(tf *opsv1alpha1.Traceflow) error {
 	c.injectedTags[tf.Status.DataplaneTag] = tf.Name
 	c.injectedTagsMutex.Unlock()
 
+	TCPSrcPort := uint16(0)
```

Comment:
nit.
I would recommend using `var X, Y, Z uint16` layout.
Also, for a local variable, please use the name starts with a lower case.

Reply:
Thanks. Updated. Since `TCP, UDP` are terms, I prefer not using `tcp, udp`. So I changed them to `dstTCPPort, dstUDPPort`, which follows style like `dstIP`.